packages/fxa-settings/src/components/Settings/Page2faChange/mocks.tsx

```diff
@@ -9,7 +9,7 @@ import {
   mockSession,
 } from '../../../models/mocks';
 import { Account, AppContext } from '../../../models';
-import Page2faChange from '.';
+import { Page2faChange } from '.';
```

Comment:
self-review comment - importing the inner page for tests and storybook (not the guarded page)